qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/CreateIdentityUITest.java

```diff
@@ -68,28 +75,50 @@ public void setUp() throws Exception {
         wakeLock = UIActionHelper.wakeupDevice(mActivity);
         mSystemAnimations = new SystemAnimations(mActivity);
         mSystemAnimations.disableAll();
+
+        //TODO TMP Fix option
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            InstrumentationRegistry.getInstrumentation().getUiAutomation().executeShellCommand(
+                    "pm grant " + InstrumentationRegistry.getTargetContext().getPackageName()
+                            + " android.permission.READ_PHONE_STATE");
```

Comment:
depends on android runtime permissons in new versions
https://developer.android.com/training/permissions/requesting.html